guides_and_docs/virt_env_Fram_with_VScode.md

```diff
@@ -0,0 +1,75 @@
+# How to use Virtual studio Code and Fram
+
+If you like to run significant bits of code, you probably want to do it on a Server. I decided to run my code on Fram, one of the Sigma2 HPCs. 
+
+I use Visual Studio Code (VS code) to connect to Fram and to run my scripts. To connect to any remote via ***ssh***, please follow the [instructions](https://code.visualstudio.com/docs/remote/ssh) on the VS code documentation. If you want to connect to Fram via VS code, use `<username>@fram.sigma2.no`, while Nird needs to know about the individual nodes. Nird: `<username>@loginX.nird.sigma2.no` where X is the node 0, 1, 2, or 3.
+
+> **_NOTE:_**  The problem is that login.nird.sigma2.no is an alias corresponding to 4 login nodes:
```

Comment:
Add: If you log in using ssh in your terminal you would use login[...] which is an alias [...]